### PR TITLE
Adds a tutorial for single/two qubit heatmaps

### PR DESCRIPTION
**Pull request by tanujkhattar**
- Adds a tutorial for single / two qubit heatmaps

### CODE REVIEW

**Review by balopat — CHANGES_REQUESTED** [review]
A lot of this tutorial Google focused. I'd split it up into two parts: 

1. Introduce a new page under `Guide` - *Visualizations* - this could have
  a.) Heatmaps 
  b.) Circuit diagrams (ASCII, SVG, etc.) 
  c.) later will have the bloch sphere, kak and others  
2. "Visualizing calibration metrics" would live under docs/tutorials/google 
 
This PR could start with just point 2.)
Also: 

- Make sure to have an entry in docs/_book.yaml for it. 
- add the authentication setup to one of the cells - otherwise this is not going to run using Quantum Engine on Colab (you can just copy-paste it from docs/tutorials/colab.ipynb)

File: docs/tutorials/google/visualizing_calibration_metrics.ipynb [hunk 1]
```diff
@@ -0,0 +1,264 @@
+{
```
Comment (balopat) [inline docs/tutorials/google/visualizing_calibration_metrics.ipynb:1]: <ul><li>visualise -&gt; visualize</li><li>Quantum Engine -&gt; Google QCS</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3880/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (balopat) [inline docs/tutorials/google/visualizing_calibration_metrics.ipynb:1]: <p>Quantum Computer -&gt; quantum processor</p><p>Quantum Engine -&gt; Google Quantum Computing Service or Quantum Engine API or <code>QuantumEngine</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3880/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (balopat) [inline docs/tutorials/google/visualizing_calibration_metrics.ipynb:1]: <p>theoretically we have the project id now from the previous cell, so you can reuse that here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3880/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

File: docs/visualization/heatmaps.ipynb [hunk 2]
```diff
@@ -0,0 +1,140 @@
+{
```
Comment (balopat) [inline docs/visualization/heatmaps.ipynb:1]: <p>visualise -&gt; visualize</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3880/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (balopat) [inline docs/visualization/heatmaps.ipynb:1]: <p>For eg -&gt; For example</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3880/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

**Review by balopat — APPROVED** [review]
Thank you!!! LGTM after the final nits

File: docs/tutorials/google/visualizing_calibration_metrics.ipynb (hunk 1 above)
Comment (tanujkhattar) [reply docs/tutorials/google/visualizing_calibration_metrics.ipynb:1]: done

Comment (tanujkhattar) [reply docs/tutorials/google/visualizing_calibration_metrics.ipynb:1]: done

File: docs/visualization/heatmaps.ipynb (hunk 2 above)
Comment (tanujkhattar) [reply docs/visualization/heatmaps.ipynb:1]: done

Comment (tanujkhattar) [reply docs/visualization/heatmaps.ipynb:1]: done

File: docs/tutorials/google/visualizing_calibration_metrics.ipynb (hunk 1 above)
Comment (tanujkhattar) [reply docs/tutorials/google/visualizing_calibration_metrics.ipynb:1]: done

**Pull request merged**